nixos/modules/services/misc/jellyfin.nix

```diff
@@ -64,14 +64,16 @@ in
         AmbientCapabilities = "";
         CapabilityBoundingSet = "";
 
-        # ProtectClock= adds DeviceAllow=char-rtc r
```

Comment:
Not so sure what to do with this one? Do I still need to pass an empty `DeviceAllow` before I make additions?

Reply:
I guess its not relevant anymore because we are already dropped ProtectClock before that.

Reply:
I would personally replace this line with `ProtectClock = true`, now. I think I remember putting that comment here because there was a conflict between `ProtectClock = true` and `DeviceAllow = ""` / `PrivateDevices = true`, but it shouldn't bother us anymore. In any case, I don't see a usecase for jellyfin to modify the hardware / system clock.

Reply:
It cannot do this anyway because the system call filter below blocks such system calls.